pandas/tests/series/methods/test_between.py

```diff
@@ -38,3 +39,12 @@ def test_between_period_values(self):
         result = ser.between(left, right)
         expected = (ser >= left) & (ser <= right)
         tm.assert_series_equal(result, expected)
+
+    def test_between_inclusive_is_boolean_string(self):
+        msg_1 = "Inclusive has to be either string of 'both','left', "
+        msg_2 = "'right', or 'neither', or a boolean value"
+        msg = msg_1 + msg_2
```

Comment:
try to write a multiline str instead of combining two single lines

Reply:
@attack68 thanks for your feedback! I tried using that originally but was thrown an error that it didn't match the regex as there's technically a `/n`. When using just a one-line variable declaration there is an error thrown that there's two many characters in one line. This is the only way I have found it to work so far.

Reply:
https://stackoverflow.com/a/10660443